Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -1,18 +1,61 @@
 {% block sonata_security_roles_widget %}
 {% spaceless %}
-    <div class="editable">
-        <h4>{{ 'field.label_roles_editable'|trans({}, "SonataUserBundle") }}</h4>
-        {{ block('choice_widget') }}
+    {% set l = labelPermission | length %}
+    {% set i = 0 %}
+    <div class="role_table">
+        <table class="table">
+            <tr>
+                <th></th>
+                {% for j, label in labelPermission %}
+                    <th {{ 'data-x=' ~ i ~ ' data-y=' ~((j % l) + 1) }}> {{ label }} </th>
```

Comment:
```twig
<th data-x="{{ i }}" data-y="{{ (j % 1) + 1 }}">
    {{- label -}}
</th>
```

Why are you adding these data attributes btw ?
I can't see any use of them in the diff...